x-pack/plugins/session_view/README.md

```diff
@@ -0,0 +1,21 @@
+# Session View
+
+Welcome to the Kibana Security Solution plugin! This README will go over getting started with development and testing.
+
+## Development
+
+## Tests
+
+### Unit tests
+
+From kibana path in your terminal go to this plugin root:
+
+```bash
+cd x-pack/plugins/session_view
+```
+
+Then run jest with:
+
+```bash
+npm run test:jest
```

Comment:
I think we should stick with yarn since the entire project uses yarn. Probably good to be consistent

Also maybe worth mentioning we can use the `-i` flag to specify folders or `-t` since `describe.only` doesn't actually do anything in jest

Reply:
You got it! 

Also, I will add a note about, the `-i` flag, if someone wants to run from the kibana root.